makeflow/src/makeflow.c

```diff
@@ -2134,7 +2134,7 @@ int main(int argc, char *argv[])
 		fprintf(stderr, "makeflow: couldn't create batch queue.\n");
 		if(port != 0)
 			fprintf(stderr, "makeflow: perhaps port %d is already in use?\n", port);
-		exit(EXIT_FAILURE);
+		goto FAILURE_LABEL;
```

Comment:
goto EXIT_WITH_FAILURE   ?

FAILURE_LABEL is not were descriptive. (All gotos go to labels...)